core/const.py

```diff
@@ -701,6 +701,7 @@ class ITLRegion(StrEnum):
     "OutcomeRef": ["OutcomeName"],
     "OutcomeData": ["SubmissionID", "ProjectID", "Outcome", "StartDate", "EndDate", "GeographyIndicator"],
     "RiskRegister": ["SubmissionID", "ProgrammeID", "ProjectID", "RiskName"],
+    "SubmissionRef": ["SubmissionID", "ReportingPeriodStart", "ReportingPeriodEnd", "ReportingRound"],
```

Comment:
This doesn't need multiple values, as SubmissionID's are unique, and should only appear in this extract once each, so the other columns will never be sorted on. So can just keep "SubmissionID"

Reply:
fixed in 328ae2f